answer/166ryoryo/JS/01/wTime.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="utf-8">
+<title></title>
+<meta name="keywords" content="">
+<meta name="description" content="">
+</head>
+<body>
+  <p id="RealtimeClockArea"></p>
+  <script lang="javascript">
+    function showClock1() {
+      const nowTime = new Date();
+      const nowHour = nowTime.getHours();
+      const nowMin = nowTime.getMinutes();
+      const nowSec = nowTime.getSeconds();
+      const msg = "現在時刻は、" + nowHour + ":" + nowMin + ":" + nowSec + " です。";
```

Comment:
```suggestion
      const msg =` 現在時刻は、${nowHour}:${nowMin}:${nowSec}です。`;
```
文字と変数の連結にはこれがオススメです
FYI: https://qiita.com/kura07/items/c9fa858870ad56dfec12